examples/benchmark/main.rs

```diff
@@ -179,10 +178,8 @@ pub fn main() {
             let factory: IDStorageFactory =
                 DStorageGetFactory().expect("Can't fer DirectStorage factor");
 
-            factory.SetDebugFlags(
-                DSTORAGE_DEBUG::DSTORAGE_DEBUG_SHOW_ERRORS
-                    | DSTORAGE_DEBUG::DSTORAGE_DEBUG_BREAK_ON_ERROR,
-            );
+            // TODO: Fix in custom emitter.csp
```

Comment:
Please move out this TODO into a github issue.

Reply:
Also reported in https://github.com/microsoft/DirectStorage/issues/38, and tried out in https://github.com/MarijnS95/direct-storage-rs/compare/windows-bindgen...custom-metadata.